audiogenerators/get_generators_linux.go

```diff
@@ -0,0 +1,7 @@
+package audiogenerators
```

Comment:
Do you need a go:build statement so this is only included on Linux?? Did you try building this on Windows?